src/ReachSets/ContinuousPost/BFFPS19/BFFPS19.jl

```diff
@@ -0,0 +1,274 @@
+        𝑂validated[:vars] = 𝒫.options[:vars]
+    else
+        𝑂validated[:vars] = 1:𝑂validated[:n]
+    end
+
+    # :partition option: use 1D blocks
+    if haskey_specified(𝒫.options, :partition)
+        𝑂validated[:partition] = 𝒫.options[:partition]
+    else
+        𝑂validated[:partition] = [[i] for i in 1:𝑂validated[:n]]
+    end
+
+    opD = 𝒫.options[:opD]
+    @assert opD isa DecomposedDiscretePost "this continuous post operator works " *
+                                       "only with DecomposedDiscretePost"
+    HS = 𝒫.options[:HS]
+    constrained_dims = constrained_dimensions(HS)
+    out_vars = opD.options[:out_vars]
+    loc_id = 𝒫.options[:loc_id]
+    temp_vars = unique([out_vars; constrained_dims[loc_id]])
+    temp_vars = get_vars(𝑂validated[:partition], temp_vars)
+    opD.options[:temp_vars] = temp_vars
+    guards_constraints = [guard(HS, trans) for trans in out_transitions(HS, loc_id)]
+    𝑂validated[:vars] = temp_vars
+    𝑂validated[:guards_proj] = [project(c, temp_vars) for c in guards_constraints]
+    𝑂validated[:blocks] = compute_blocks(𝑂validated[:vars], 𝑂validated[:partition])
+
+    # :block_options_init & :block_options_iter options:
+    # set default according to :partition
+    if !haskey_specified(𝒫.options, :block_options_init)
+        𝑂validated[:block_options_init] =
+            compute_default_block_options(𝑂validated[:partition])
+    end
+    if !haskey_specified(𝒫.options, :block_options_iter)
+        𝑂validated[:block_options_iter] =
+            compute_default_block_options(𝑂validated[:partition])
+    end
+
+    # Input -> Output variable mapping
+    𝑂validated[:inout_map] = inout_map_reach(𝑂validated[:partition], 𝑂validated[:blocks], 𝑂validated[:n])
+
+    if 𝑂validated[:project_reachset]
+        𝑂validated[:output_function] = nothing
+    else
+        𝑂validated[:output_function] = 𝑂validated[:projection_matrix]
+    end
+
+    return 𝑂validated
+end
+
+"""
+    post(𝒫::BFFPS19, 𝑆::AbstractSystem, 𝑂::Options)
+
+Calculate the reachable states of the given initial value problem using `BFFPS19`.
+
+### Input
+
+- `𝒫` -- post operator of type `BFFPS19`
+- `𝑆` -- sytem, initial value problem for a continuous ODE
+- `𝑂` -- algorithm-specific options
+"""
+function post(𝒫::BFFPS19, 𝑆::AbstractSystem, 𝑂_input::Options)
+    𝑂 = TwoLayerOptions(merge(𝑂_input, 𝒫.options.specified), 𝒫.options.defaults)
+
+    @assert 𝑂[:mode] == "reach" "the mode $(𝑂[:mode]) is not supported"
+
+    info("Reachable States Computation...")
+    @timing begin
+        Rsets = reach_mixed(𝑆, 𝑂)
+
+        info("- Total")
+    end
+
+    # Projection
+    if 𝑂[:project_reachset]
+        info("Projection...")
+        RsetsProj = @timing project(Rsets, 𝑂)
+    else
+        RsetsProj = Rsets
+    end
+
+    return ReachSolution(RsetsProj, 𝑂_input)
+end
+
+function get_vars(partition, vars)
+    result = Vector{Int}()
+    for ur in partition
+        if any(v ∈ vars for v in collect(ur))
```

Comment:
```suggestion
        if any(v ∈ ur for v in vars)
```
This check is more efficient. Assuming that `vars` is sorted, one could do this even more efficiently, but we can keep it simple here.